index.js

```diff
@@ -0,0 +1,21 @@
+// Modal functionality
+let modal = document.getElementById("instructionsModal");
+let btn = document.getElementById("instructionsModalBtn");
+let span = document.getElementsByClassName("close")[0];
+
```

Comment:
Possibly remove line breaks? It adds a lot of white space, but makes it easy to read too.